testsuite/tests/conftest.py

```diff
@@ -377,12 +377,10 @@ def _custom_user(params, autoclean=True):
 
 
 @pytest.fixture(scope="session")
-def staging_gateway(request, testconfig, configuration):
+def default_staging_gateway(request, testconfig, configuration, openshift):
```

Comment:
I didn't get concept of `default_staging_gateway`, don't understand, considering it pointless (explanation will fix that)
Also recurring clones of `def staging_gateway` aren't clear to me

Reply:
The concept was that in many tests we duplicated the code from main conftest when we needed default gateway, since the code wasn´t trivial in previous iterations of this PR, I wanted to simplify it and have less copy-paste there, however through many iterations, the complicated initialization that once was in `staging_gateway` is now gone and so this is now pointless and I will remove it.